src/shifter.c

```diff
@@ -188,11 +188,11 @@ int main(int argc, char **argv) {
 
         if (entry != NULL) {
             opts.args[0] = strdup(entry);
-            if (imageData.workdir != NULL) {
-                opts.workdir = strdup(imageData.workdir);
-            }
         }
     }
+    if (imageData.workdir != NULL && opts.workdir == NULL) {
```

Comment:
I don't think the changes the current default behavior. Right now Shifter uses the CWD. The only way to use the workforce is pass the entry point. So I guess it does change that behavior. We could add that back to that block.